lib/Twig/Environment.php

```diff
@@ -213,24 +216,37 @@ public function isStrictVariables()
     }
 
     /**
-     * Gets the cache directory or false if cache is disabled.
+     * Gets the current cache implementation.
      *
-     * @return string|false
+     * @return Twig_CacheInterface|string|false A Twig_CacheInterface implementation,
```

Comment:
the return string or false looks wrong IMO, as these are converted to `Twig_CacheInterface` anyway in the only entry point for this property (which is `setCache`)